python/lsst/ts/externalscripts/base_make_calibrations.py

```diff
@@ -599,6 +605,186 @@ async def verify_calib(self, image_type, job_id_calib, exposure_ids_dict):
 
         return response
 
+    async def check_verification_stats(self, image_type, job_id_calib, job_id_verify):
```

Comment:
General comment regarding the return value of this method. Wouldn't it be best to return a single dictionary with all the information stored in it?